### DISCUSSION TITLE
Contact sales

### DISCUSSION THREAD

**Discussion by amnonbb** [emqx/emqx#6253] [General]
How march time take to be answered from sales@emqx.io?

**Comment by zmstone** [comment]
hi @amnonbb 
We are usually quite responsive, but some emails may get wrongly auto filtered as spam
Did you write to sales@emqx.io?
If you let me know when it was sent from whom, I can ask my colleague to have a look.

**Comment by amnonbb** [comment]
We use form in emqx.com and after 2-3 days we wrote to sales@emqx.io.
Mail sent from orders@kab.info.

**Comment by zmstone** [comment]
Sorry, it fell between the chairs. 
We will reply asap.
